dashboard/app/models/script.rb

```diff
@@ -1724,7 +1724,7 @@ def summarize_header
       isHocScript: hoc?,
       student_detail_progress_view: student_detail_progress_view?,
       age_13_required: logged_out_age_13_required?,
-      is_csf: csf?
+      is_csf: csf? || csc?
```

Comment:
Ideally we would update is_csf to something that better captures this but in order to keep this change prior to next week I thought this was safer.